daft/dataframe/dataframe.py

```diff
@@ -879,15 +832,18 @@ def explode(self, *columns: ColumnInputType) -> "DataFrame":
             DataFrame: DataFrame with exploded column
         """
         parsed_exprs = self.__column_input_to_expression(columns)
-        return DataFrame(logical_plan.Explode(self._plan, parsed_exprs))
+        builder = self._builder.explode(parsed_exprs)
+        return DataFrame(builder)
 
     def _agg(
         self, to_agg: List[Tuple[ColumnInputType, str]], group_by: Optional[ExpressionsProjection] = None
     ) -> "DataFrame":
         exprs_to_agg: List[Tuple[Expression, str]] = list(
             zip(self.__column_input_to_expression([c for c, _ in to_agg]), [op for _, op in to_agg])
         )
-        builder = AggregationPlanBuilder(self._plan, group_by=group_by)
+        # TODO(Clark): Port AggregationPlanBuilder to new LogicalPlanBuilder once Charles has merged in his PR.
```

Comment:
@xcharleslin I opted to not touch this for now in order to not conflict with your `Agg` PR!